docs/application/dotnet/guides/security/privacy-related-permissions.md

```diff
@@ -227,7 +232,7 @@ To check whether an application has permission to use a privilege, and to reques
         }
       ```
 
-> **Note**
+> [!NOTE]
 >
```

Comment:
@yl33  
Request you to check the following: 

Global, 
In whole file, can we remove extra space between: 
> [!NOTE]
> text....

Reply:
Removed empty line after note tag.